### ISSUE TITLE
[Feature Request]: - Your request

### ISSUE THREAD

**Issue by krishnakhadka200416** [deel-ai/influenciae#23]
### Module

Common

### Feature Request

Can we calculate IF for simple models such as logistic regression?

### Version

v0.1.0

### Environment

```markdown
- OS:
- Python version:
- Tensorflow version:
- Packages used version:
```

**Comment by Agustin-Picard** [comment]
Yes, of course! A logistic regression is simply a `model = Sequential([Input(shape=input_shape), Dense(n_classes, activation='softmax')])` with a `unreduced_loss = CategoricalCrossentropy(from_logits=False, reduction=Reduction.NONE)`, so you just create your `InfluenceModel` taking your only layer's weights (`influence_model = InfluenceModel(model, start_layer=-1, loss=unreduced_loss)`) and it should work just fine.

If you're doing a single class regression, just adjust your loss and activation function and it should work.
